tests/test_hinfo.py

```diff
@@ -0,0 +1,81 @@
+import os
+import shutil
+import sqlite3
+import tempfile
+from pathlib import Path
+
+import pytest
+import yaml
+from lsst.consdb import hinfo, utils
+from lsst.resources import ResourcePath
+from sqlalchemy import MetaData, Table, select
+
+
+@pytest.fixture
+def tmpdir(scope="module"):
+    os.environ["INSTRUMENT"] = "LATISS"
+    tmpdir = Path(tempfile.mkdtemp())
+    try:
+        yield tmpdir
```

Comment:
Can the yield actually ever fail?  I think you could remove the `try: finally:` (here and in the other test).